federation/v1/importedserviceset_types.go

```diff
@@ -23,29 +23,29 @@ import (
 // +genclient
 // +k8s:deepcopy-gen:interfaces=k8s.io/apimachinery/pkg/runtime.Object
 
-// ServiceImports is the Schema for configuring imported services.  The name of
-// the ServiceImports resource must match the name of a MeshFederation resource
+// ImportedServiceSet is the Schema for configuring imported services.  The name of
+// the ImportedServiceSet resource must match the name of a MeshFederation resource
```

Comment:
s/MeshFederation/ServiceMeshPeer/

Reply:
done.  also tried to remove generic "mesh" terms with "peer.: